geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -1653,6 +1653,14 @@
   public static final String LIST_FUNCTION__NO_FUNCTIONS_FOUND_ERROR_MESSAGE = "No Functions Found";
 
   public static final String LIST_GATEWAY = "list gateways";
+  public static final String LIST_GATEWAY__SHOW_SENDERS_ONLY = "senders-only";
+  public static final String LIST_GATEWAY__SHOW_SENDERS_ONLY__HELP =
+      "Display gateway senders only.";
+  public static final String LIST_GATEWAY__SHOW_RECEIVERS_ONLY = "receivers-only";
+  public static final String LIST_GATEWAY__SHOW_RECEIVERS_ONLY__HELP =
+      "Display gateway receivers only.";
+  public static final String LIST_GATEWAY__ERROR_ON_SHOW_PARAMETERS =
+      "Options --senders and --receivers cannot be used together.";
```

Comment:
In order to keep the messages consistent with the parameter names, this should read `Options --senders-only and --receivers-only cannot be used together.`